src/AdamWathan/Form/Elements/Checkbox.php

```diff
@@ -76,11 +76,14 @@ protected function setChecked($checked = true)
 
     protected function checkBinding()
     {
-        $currentValue = $this->getAttribute('value');
+        $currentValue = (string) $this->getAttribute('value');
```

Comment:
Sure man.  So at first `===` made sense to me, but then this happened.  I was minding my own business trying to implement simple radios with binary values...
```
$form->radio('published', 1)->render();
$form->radio('published', 0)->render();
```

In context of Laravel/Eloquent, these values are not cast to integers unless I setup an accessor on my model and intval() the thing.  However, the value stored on the field object is stored as integer.  So by default the old input provider shows the user's selected value as string, but the value on the element object is integer, causing this  `1 === "1"` to not bind the old input.

The problem is, we can't simple `1 == "1"`.  Imagine this scenario...
```html
<input type="radio" name="published" value="0">
```

If there is no old value ($oldValue on the field object is then `null`), then doing a `0 === null` equates to true, and the radio is checked, as if it were old input!  So in this case we DO WANT `===`, but in the first case we don't want `===`.

Casting everything to string before comparing values solves both problems.  TL;DR... The HTML value attribute on checkboxes/radios know nothing about an integer, or anything about data types for that matter.  All it knows is string.  If we cast everything to string first, we get a more accurate comparison of old input.